endpoint/endpoint_diagnoseComp/endpoint_diagnose.c

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2019-2020 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+
+#include "endpoint_diagnose.h"
+
+#include "legato.h"
+#include "le_gnss_interface.h"
+#include "le_mrc_interface.h"
+
+COMPONENT_INIT {
+
+  char time_str[30];
+  size_t copied_bytes = 0;
+  int32_t latitude = 0, longitude = 0, accuracy = 0;
```

Comment:
Is it better to include the stdint.h header?

Reply:
It has been included in `legato.h`
Ref: https://docs.legato.io/latest/legato_8h_source.html